config/initializers/admins.rb

```
@@ -0,0 +1,4 @@
+module Gemcutter
+  mattr_accessor :admins
+  self.admins = Application.config_for("admins")["emails"]
+end
```

Nice.  Do we want a `fetch` here for fail fast?